contracts/axelar-gateway/src/contract.rs

```diff
@@ -190,27 +197,31 @@ impl AxelarGatewayInterface for AxelarGateway {
             .persistent()
             .set(&DataKey::RotationExecuted(data_hash), &true);
 
-        auth_module.rotate_signers(&signers, &true);
+        let operator: Address = Self::operator(&env);
```

Comment:
this is a breaking change. Add this in a separate PR. Specifically, need to mimic what EVM gateway does here

Reply:
Agreed. Removed `operator` related lines and merged PR.